target/linux/ramips/mt76x8/base-files/etc/board.d/01_leds

```diff
@@ -53,6 +53,7 @@ tama,w06)
 	ucidef_set_led_wlan "wlan" "WLAN" "$boardname:green:wlan" "phy0tpt"
 	;;
 tplink,archer-c20-v4|\
+tplink,archer-c20-v5|\
```

Comment:
Same here: Can phyXtpt triggers used in DTS?

Reply:
Yes.